docs/modules/ROOT/pages/integrations/database.adoc

```diff
@@ -0,0 +1,69 @@
+= Database Integrations
+
+== Overview
+
+KubeArchive needs a database to store the resources to make them retrievable.
+KubeArchive currently supports integration with PostgreSQL.
+MariaDB integration is in progress.
+
+[NOTE]
+====
+
+KubeArchive integration test suite runs with PostgreSQL 16.6
+
+====
+
+== Configuration and Customization
+
+KubeArchives offers a [filename]`kubearchive.sql` file with the DDL to create
+the schema as part of the installation.
+
+[IMPORTANT]
+====
+The provided schema includes indexes.
+
+KubeArchive users should review and customize the indexes
+if needed based on the expected queries and the amount and
+the distribution of the expected archived data.
+====
+
+Specific instructions for the configuration of the PostgreSQL Database are available in
+xref:/getting-started/configuration.adoc#_install_and_configure_kubearchive[Install and Configure KubeArchive].
```

Comment:
The instructions are not in the `configuration.adoc` document, link directly to the `installation.adoc` file.

Reply:
Right! It's fixed now